components/edit-collective/sections/PaymentReceipts.js

```diff
@@ -0,0 +1,270 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gql, useQuery } from '@apollo/client';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
```

Comment:
Any reason for adding `dayjs` here?
For consistency, I would rather keep using `momentjs`.